### ISSUE TITLE
feat(reddit): De-tiktokify video threads

### ISSUE THREAD

**Issue by jordan1776** [ReVanced/revanced-patches#787]
### Application

reddit

### Issue

I do not like the way video threads on the official reddit app are handled. I believe they should follow the same format as link threads and image threads. Especially with the way gesturing down plays another video which I find not in the spirit of what reddit is about. It makes it more like tiktok or youtube shorts, which isn't what I go to reddit for.

### Patch

Changes to the video player:
Gesturing up should always discard the player (just like with images and gesturing down with videos).
Have a toggle setting to move or disable all video interactions to three button menu (upvote, downvote, share, award)
Video speed control? (minor improvement, not partially necessary)

Changes to the comment thread:
The thread should display like a link/image thread (remove the strange half and half drawer view)
Video should be a thumbnail (no autoplay).
Enable thread to thread left/right swiping, like normal comment threads.

Basically talking the view from the image/linked thread; and using the thumbnail with 'play' icon in place of the image should be enough. And tapping the thumbnail would play the video.

### Motivation

I feel that the way videos are treated on the official app goes against the ethos of reddit. The changes I suggest would make the app more inline with a user submitted link aggregator site rather than a social media app. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Over time, a lot of feature requests have been made, which is why I am closing some according to the [contribution guidelines](https://github.com/ReVanced/revanced-patches/blob/main/CONTRIBUTING.md#-submitting-a-feature-request):

> Requests can be accepted or rejected at the discretion of maintainers of ReVanced Patches

This feature request has been closed for one or more of the following reasons:

- Stale for too long
- Low priority
- Maintainers un-interested
- Too complex to realize

Regardless, anyone who comes across this feature request can PR it. Thanks for the feature request!
